app.cr

```diff
@@ -34,12 +34,13 @@ end
 
 def fetch_filter(context)
   filter = context.params["filter"]? || ""
-  filter.strip
+  filter = filter.strip.downcase unless filter.empty?
```

Comment:
Good point. `try` is awesome. Here is how it might look now:
```ruby
def fetch_filter(context)
  context.params["filter"]?.try(&.strip.downcase) || ""
end
```